arborist/policy.go

```diff
@@ -288,10 +298,48 @@ func (policy *Policy) deleteInDb(tx *sqlx.Tx) *ErrorResponse {
 	return nil
 }
 
-func (policy *Policy) overwriteInDb(tx *sqlx.Tx) *ErrorResponse {
-	errResponse := policy.deleteInDb(tx)
+func (policy *Policy) updateInDb(tx *sqlx.Tx) *ErrorResponse {
+	// We do not allow updates to policy name (or id).
+
+	errResponse := policy.validate()
 	if errResponse != nil {
 		return errResponse
 	}
-	return policy.createInDb(tx)
+
+	var policyID int
+	stmt := "SELECT id FROM policy WHERE name = $1"
```

Comment:
see if you need `SELECT ... FOR UPDATE` here,  or merge it with the `UPDATE` below with `RETURNING id`